.values/README.md

```diff
@@ -42,6 +42,8 @@ There are two ways to configure these: manually, or with the Otomi Console UI.
 
 It's easiest to start by editing the basic configuration settings in the files themselves. It also helps to understand what is going on. (The bulk of the work is setting up and harvesting the credentials for all the Azure service principals, Google service accounts and AWS secrets and keys.)
 
+Initially `secrets.*.yaml` files encryption is enabled. You can disable it only once, before calling `otomi commit` command, by removing `.sops.yaml` file. You can always enable encryption later by defining `.sops.yaml` file.
+
```

Comment:
Just mention that the existence of the sops file makes it encrypt. "So please remove it if you don't want encryption straight away."